compliance_checker/imos/imos.py

```diff
@@ -323,7 +323,7 @@ def check_geospatial_vertical_min_max(self, dataset):
                 geospatial_lat_min = getattr(dataset.dataset, "geospatial_vertical_min", None)
```

Comment:
@ggalibert Need to change `geospatial_lat_min` to `geospatial_vertical_min` here too!
Sorry... just realised the code was actually checking the correct value (the above line is reading the value of the geospatial_vertical_min attribute), it was just using a very confusing variable name to store it!